packages/react/src/icon/icons/ChevronYIcon.tsx

```diff
@@ -0,0 +1,10 @@
+import { Fragment } from 'react';
+import { createIcon } from '../Icon';
+
+export const ChevronYIcon = createIcon(
```

Comment:
Hey @nathsimpson , could we please rename this? I've never heard of this type of icon being referred to as "Chevron Y" before.

Maybe `ChevronsUpDown`? That was the name proposed in feather icons, see here for more details: https://github.com/feathericons/feather/issues/234

Reply:
Good thought. Fixed 👍